src/components/Section.js

```diff
@@ -53,6 +53,7 @@ class Section extends Component {
       td: {
         fontSize: 0,
         verticalAlign: mjAttribute('vertical-align'),
+		padding: mjAttribute('padding'),
```

Comment:
Good ! Can you just fix your editor to put space instead of tabs so this can be merged ? Thanks !